modules/exploits/windows/smb/ms17_010_eternalblue_win8.rb

```diff
@@ -0,0 +1,951 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'ruby_smb'
+require 'ruby_smb/smb1/packet'
+require 'rubyntlm'
+require 'net/ntlm'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
```

Comment:
```suggestion
  Rank = AverageRanking
```

https://github.com/rapid7/metasploit-framework/pull/15182/files#r632091672